content/backmatter.tex

```diff
@@ -181,6 +187,16 @@ \chapter{Undefined Behavior in OpenSHMEM}\label{sec:undefined}
 \end{itemize}
 \tabularnewline
 \hline
+Concurrently using active-set-based and team-based collectives & Concurrently
+using the default team on an active-set-based collective with the default
+team in a team-based collective, for instance inside a parallel region,
+results in undefined behavior.\tabularnewline
+\hline
+Destroying a team with unfreed private contexts & Before destroying a given
+team, the user is responsible for destroying all contexts within that team
+having the \LibConstRef{SHMEM\_CTX\_PRIVATE} option enabled, otherwise, the
+behavior is undefined.\tabularnewline
```

Comment:
```suggestion
Destroying a team with unfreed private contexts & Before destroying a given
team, the user is responsible for destroying all contexts created from that team
with the \LibConstRef{SHMEM\_CTX\_PRIVATE} option enabled; otherwise, the
behavior is undefined.\tabularnewline
```